example/dj/apps/app/tests/fieldsets.py

```diff
@@ -0,0 +1,80 @@
+from unittest.case import TestCase
+
+from germanium.tools import assert_true, assert_false, assert_equal, assert_is_none
+
+from pyston.utils import rfs
+
+
+class FieldsetsTestCase(TestCase):
+
+    def test_create_rfs_from_list(self):
+        assert_equal(str(rfs(('a', 'b', 'b__c', 'b__g', ('d', ('e__f',))))), 'a,b(c,g),d(e(f))')
+        assert_equal(str(rfs(('a', 'b', ('d', ('e__f',)), ('b', ('c',))))), 'a,b(c),d(e(f))')
+        assert_equal(str(rfs(())), '')
+
+    def test_rfs_append(self):
+        fieldset = rfs(('a', 'b', 'b__c', 'b__g', ('d', ('e__f',))))
```

Comment:
jak si můžete všimnout definice fieldsetů je dvojí buď pomocí listu

(a, (b,))

nebo stringem jako se tefinují list_display v is-core

a__b